src/drivers/Odnoklassniki.php

```diff
@@ -0,0 +1,40 @@
+<?php
+/**
+ * @link https://github.com/yiimaker/yii2-social-share
+ * @copyright Copyright (c) 2017 Yii Maker
+ * @license BSD 3-Clause License
+ */
+
+namespace ymaker\social\share\drivers;
+
+use ymaker\social\share\base\AbstractDriver;
+
+/**
+ * Driver for Ok.ru
+ *
+ * @link https://ok.ru
+ *
+ * @author Petro Sokolnykov <info@xyz.net.ua>
+ * @since 1.0
+ */
+class Odnoklassniki extends AbstractDriver
+{
+    /**
+     * @inheritdoc
+     */
+    protected function processShareData()
+    {
+        $this->url = static::encodeData($this->url);
+        $this->title = static::encodeData($this->title);
+        $this->description = static::encodeData($this->description);
+        $this->imageUrl = static::encodeData($this->imageUrl);
+    }
+
+    /**
+     * @inheritdoc
+     */
+    protected function buildLink()
+    {
+        return 'https://connect.ok.ru/offer?url={url}&title={title}&description={description}&imageUrl={imageUrl}';
```

Comment:
For better readability please refactor this like that https://github.com/yiimaker/yii2-social-share/blob/master/src/drivers/LinkedIn.php#L49